### DISCUSSION TITLE
Improvements on UI

### DISCUSSION THREAD

**Discussion by lcfd** [Forceu/Gokapi#78] [Ideas]
Hi!

Gokapi looks like a **very** good project. 
Thank you!

Do you have any plans to improve the UI? 
It is a bit unripe and may not be appealing to use.

**Comment by Forceu** [comment]
Unfortunately I do not have a lot of experience in designing, however I would gladly approve any pull request in that matter.

**Reply by lcfd** [reply]
Sure! 

It was just a humble question to understand the status of the project, and its future developments, from that point of view.
If there will be an opportunity I'll take a look 😉

**Reply by Forceu** [reply]
If you have any experience, you find the static files and templates in /internal/webserver/web/ :)
